microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 
+
+## Background
+The WebView2 team tracks data to help understand and prioritize our customers' requirements, diagnose and debug issues, and overall improve WebView2 as a product.

```

Comment:
```suggestion
The WebView2 team tracks data to help understand and prioritize developers' requirements, diagnose and debug issues, and improve WebView2.
```

Reply:
I will change customers to developers. I'm thinking to leave out the line intending this for WebView2 developers if devs want to share this with their end-users but will be more clear.